### PR TITLE
[docs] feat: add wan2.1 training guide doc and mock dataset generator

### PR DESCRIPTION
**Pull request by zbian99**
### What does this PR do?

1. Add wan2.1training guide doc

2. Since end-to-end training for the **wan2.1 i2v** model is not yet supported in main branch, we will construct random tensors to conduct test training.

3. update npu support infomation

### How to use mock dataset generator?

follow training guide in the doc.

### How to check the document effect?

Pull the code to your local machine and execute sphinx-build -b html docs docs/_build.

### CODE REVIEW

**Comment by zbian99** [comment]
label ascend

File: docs/examples/wan2.1.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+(examples-wan2.1)=
+
+# Wan2.1-I2V training guide
+
+## Download model
+
+```python
```
Comment (Crystal-jiang) [inline docs/examples/wan2.1.md]: It should be shell

File: docs/examples/wan2.1.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+pip install "huggingface_hub[cli]"
+huggingface-cli download Wan-AI/Wan2.1-I2V-14B-480P-Diffusers \
+--local-dir ./Wan2.1-I2V-14B-480P-Diffusers
```
Comment (Crystal-jiang) [inline docs/examples/wan2.1.md]: VeOmni has a ready-to-use download script — you can refer to the code below.
```shell
python3 scripts/download_hf_model.py \
    --repo_id Wan-AI/Wan2.1-I2V-14B-480P-Diffusers \
    --local_dir .
```

**Comment by Crystal-jiang** [comment]
add Ascend label

File: docs/examples/generate_wan_dataset.py [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+import torch
+
+
+# --- Fixed Architectural Constants (Internal Only) ---
+FIXED_C = 16
+FIXED_L_TEXT = 512
+FIXED_D_TEXT = 4096
+FIXED_D_CLIP = 1280
+FIXED_C_Y = 20
+
```
Comment (onehaitao) [inline docs/examples/generate_wan_dataset.py:22]: Add some annotations to explain the meaning

File: docs/examples/wan2.1.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+```shell
+bash train.sh tasks/omni/train_wan.py configs/dit/wan_sft.yaml \
+--model.model_path Wan2.1-I2V-14B-480P-Diffusers/transformer
+```
+
+## Start training on NPU
+
+```shell
+bash train.sh tasks/omni/train_wan.py configs/dit/wan_sft.yaml \
+--model.model_path Wan2.1-I2V-14B-480P-Diffusers/transformer \
```
Comment (Crystal-jiang) [inline docs/examples/wan2.1.md]: Consider adding indentation — the GPU section above should be treated the same way.
bash train.sh tasks/omni/train_wan.py configs/dit/wan_sft.yaml \
    --model.model_path Wan2.1-I2V-14B-480P-Diffusers/transformer \
    --train.init_device npu

File: docs/examples/generate_wan_dataset.py (hunk 3 above)
Comment (zbian99) [reply docs/examples/generate_wan_dataset.py:22]: Done

File: docs/examples/wan2.1.md (hunk 1 above)
Comment (zbian99) [reply docs/examples/wan2.1.md]: done

File: docs/examples/wan2.1.md (hunk 2 above)
Comment (zbian99) [reply docs/examples/wan2.1.md]: done

File: docs/examples/wan2.1.md (hunk 4 above)
Comment (zbian99) [reply docs/examples/wan2.1.md]: done
